### PR TITLE
Make mlp_explicit.py a unit test

### PR DESCRIPTION
**Pull request by TuanNguyen27**

### CODE REVIEW

**Comment by TuanNguyen27** [comment]
@googlebot I fixed it.

**Comment by TuanNguyen27** [comment]
hi @andsteing , i'm trying to tackle #525 , wondering if you could take a look. I don't think my test is actually running. Thank you!

File: tests/linen/linen_mlp_test.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    # Assuming we're in a jit, should use no FLOPs -- "just shape inference".
+    self.__call__(jnp.zeros((1, self.in_features, )))
+
+class MLP(Module):
+  def setup(self):
+    self.dense1 = DenseExplicit(in_features=3, features=2)
+    self.dense2 = DenseExplicit(in_features=2, features=1)
+
+    # explicit instances are materialized immediately at init
+    # pprint(self.dense2.variables)
```
Comment (avital) [inline tests/linen/linen_mlp_test.py:46]: Please replace this comment with an assertion for the keys and shapes of `self.dense2.variables`

**Review by avital — COMMENTED** [review]
Thanks @TuanNguyen27! I added one comment.

File: tests/linen/linen_mlp_test.py (hunk 1 above)
Comment (avital) [reply tests/linen/linen_mlp_test.py:46]: (or actually, we can keep the comment because it's clarifying, but add an assertion)

**Comment by avital** [comment]
Hi @TuanNguyen27 do you have time to take a look at the review?

**Comment by avital** [comment]
I'll close this for now as there hasn't been activity recently, feel free to reopen if it's ready for submission.

**Pull request closed without merging**